frontend/src/components/common/footer/index.css

```diff
@@ -78,6 +78,14 @@
   text-align: center;
 }
 
+.footer-line {
+  border: 0.7px solid white;
+  margin-left: 220px;
+  margin-right: 175px;
```

Comment:
this looks really awesome at larger screen sizes but not so ideal at a smaller screen size
![image](https://user-images.githubusercontent.com/1637481/66608601-8cab0100-eb7c-11e9-9e71-c56abd84d124.png)

There are a couple of ways to solve this -> we could do different styles at different screen widths 

but I think the best is to have the `margin-left` and `margin-right` be %s. Playing with it, I think 10% is best.

Reply:
I updated the code to include percentages instead of pixel sizes.